sklearnex/tests/test_memory_usage.py

```diff
@@ -79,28 +79,30 @@ def dataframe_f(x, y):
 ]
 
 
-def gen_clsf_data():
+def gen_clsf_data(estimator_name):
```

Comment:
It's better to count algorithm extra memory usage outside of data generation function